### DISCUSSION TITLE
page list filter of articles by date

### DISCUSSION THREAD

**Discussion by bartart3d** [automadcms/discussions#6] [Q&A]
Hello,

I'm trying to create two page lists, defined by how recent the articles are. I don't see how I can achieve it in Automad.
One page list would have the title 'Now' which contains the pages written the latest 12 months
The other page list would have the title 'Then' which contains the pages before those 12 months
I know I should use a filter to achieve that, but I have no idea how to implement it.
If someone here can point me in the right direction to get this to work...  :-)

**Comment by marcantondahmen** [comment]
Hi, for this kind of page lists you can use plain PHP in a template or create a custom extension that generates the page lists. In both cases the implementation must be in PHP.
